pkg/report/testdata/linux/report/681

```diff
@@ -0,0 +1,30 @@
+TITLE: KMSAN: uninit-value in net_tx_action
+ALT: KMSAN origin in netlink_rcv_skb
```

Comment:
netlink_rcv_skb also looks like a common function used in several netlink protocols.
The protocol-specific function here is rtnetlink_rcv.
So perhaps we want to skip netlink_rcv_skb as well, or it may lead to the same problem in future.